assembl/tests/views_tests/test_discussion_export.py

```diff
@@ -0,0 +1,31 @@
+from graphql_relay import from_global_id
+
+from assembl import models
+from assembl.views.api2.discussion import thread_csv_export
+from assembl.views.traversal import InstanceContext
+from freezegun import freeze_time
+
+
+@freeze_time("2018-3-8")
+def test_thread_csv_export(discussion, request, thread_phase_root_idea, user_language_preference_en_cookie,
+                           top_post_in_thread_phase, second_post_in_thread_phase, third_post_in_thread_phase):
+    request.GET = {}
+    assert thread_phase_root_idea.message_view_override == 'thread'
+    request.context = InstanceContext(None, discussion)
+    header, results = thread_csv_export(request)
+    assert len(results) == 3
+    assert results[0]['Th\xc3\xa9matique niveau 1'] == 'Understanding the dynamics and issues'
```

Comment:
You could add `# -*- coding: utf-8 -*-` at top of the page, and make this key `u'Thématique niveau 1'` if you want, should work the same way...

Reply:
fixed